test/sql/test_function/T/test_named_argments

```diff
@@ -0,0 +1,12 @@
+-- name: test named arguments for table functions
+
+select * from TABLE(generate_series(start=>2, end=>5));
```

Comment:
You need to add tests to cover normal cases.
I see all cases are fail cases.

Reply:
added